### DISCUSSION TITLE
Linkerd multicluster gateway

### DISCUSSION THREAD

**Discussion by Vamsi0473** [linkerd/linkerd2#10408] [General]
Hi Team, I am working to establish multi cluster communication between EKS clusters in order to achieve this I have followed below steps
Installed 2.9.5 linkerd and linkerd multicluster (there is a specific to install 2.9.5 for replicating the version we had in production)
Deployed gateway and link components
Tried to service mirror service A from cluster A to cluster B which is successful but when I try make call to mirrored service A from cluster B it is throwing me 503 error
I checked service mirror logs and can see following error
 time="2023-02-27T22:30:43Z" level=warning msg="Gateway returned unexpected status 503. Marking as unhealthy" probe-key=cluster A
I am stuck and don't know how to proceed I check all the cluster ELB,endpoints, roles,clusterrole bindings, role and role bindingds everything is present. Can someone please help in how to resolve this error

**Comment by wmorgan** [comment]
Linkerd 2.9.5 is no longer supported. You could check one of the [commercial support providers for Linkerd](https://linkerd.io/support-training/) which might be able to assist you.
